frontends/concrete-python/concrete/fhe/compilation/compiler.py

```diff
@@ -537,5 +537,9 @@ def compile(
         finally:
             self.configuration = old_configuration
             self.artifacts = old_artifacts
+            if self.graph and len(self.graph.output_nodes) > 1:
+                msg = "Multiple outputs are not supported"
+                graph_text = circuit.graph.format(highlighted_result=[msg], show_bounds=False)
+                raise RuntimeError(f"{msg}\n\n{graph_text}")
```

Comment:
Let's be consistent with the rest of the errors:
```
        highlighted_result = "multiple outputs are not supported"
        message = "Function you are trying to compile cannot be compiled\n\n" + graph.format(
            highlighted_result=highlighted_result,
        )
        raise RuntimeError(message)
```

Reply:
done